docs/ert/reference/configuration/keywords.rst

```diff
@@ -281,27 +315,22 @@ wherein coeff_priors
 
 ::
 
-        a UNIFORM 0 1
-        b UNIFORM 0 2
-        c UNIFORM 0 5
-        d UNIFORM 0 2
-        e UNIFORM 0 5
-
+        b UNIFORM 0 1
+        c UNIFORM 0 2
+        d UNIFORM 0 5
 
-the :ref:`GEN_KW <gen_kw>` group COEFFS would remain, but the values would be read from the design matrix while update is disabled for COEFFS.
-Notice that this requires a full overlap of parameter names.
-The case with only a partial overlap will result in a validation error.
-An example of a partial overlap is when GEN_KW contains parameters a,b,c,d,e and the design matrix contains a,b,c,d.
-Additionally, the overlap needs to be present only in a single GEN_KW group, which means that having for instance:
 
+the :ref:`GEN_KW <gen_kw>` group COEFFS would remain, but the values (for b, c and d) would be read from the design matrix
+and the update will be disabled. In this case the final set of parameters (for example in parameters.txt in real==0) would be:
 ::
 
-        GEN_KW COEFFS1 coeff_priors_1 -- defining parameters a,b,c
-        GEN_KW COEFFS2 coeff_priors_2 -- defining parameters d,e
-
-would also yield validation error, even the union of parameters overlaps the design matrix parameters.
-In such cases consider to comment out GEN_KW definitions and thus only the design parameters will be used.
+        DESIGN_MATRIX:a=1
+        COEFFS:b=1
+        COEFFS:c=2
+        COEFFS:d=0
+        DESIGN_MATRIX:e=1
 
+So the genkw would keep its name (COEFFS) and the design matrix parameters would be prefixed with DESIGN_MATRIX.
 
```

Comment:
"In case there is no overlap with a GEN_KW group, the GEN_KW group will be sampled normally." This sentence is now a bit unclear, also if there is overlap with the group, but the GEN_KW COEFF param is not present in the design matrix, it will be sampled and updated.

I think perhaps the example should have a GEN_KW COEFFS parameter that is _not_ in the design matrix too, and that final sentence changed to e.g. "GEN_KW group parameters that are not found in the design matrix will be sampled normally" or something?

Reply:
> "In case there is no overlap with a GEN_KW group, the GEN_KW group will be sampled normally." This sentence is now a bit unclear, also if there is overlap with the group, but the GEN_KW COEFF param is not present in the design matrix, it will be sampled and updated.

I agree, maybe I can just implement your suggestion:
"GEN_KW parameters that are not found in the design matrix will be sampled normally".

> I think perhaps the example should have a GEN_KW COEFFS parameter that is not in the design matrix too

It's already too long. But maybe I can just add another param to coeffs_prior `f` and emphasize that this one will be sampled?

Reply:
Another iteration:

<img width="760" height="613" alt="image" src="https://github.com/user-attachments/assets/d9ea2ba1-0e22-4192-8bf5-d95cb8ce5931" />